lib/mysql_rewinder.rb

```diff
@@ -56,7 +56,10 @@ def reset_inserted_tables
     end
 
     @inserted_tables = Set.new
-    FileUtils.rm(Dir.glob(@inserted_table_record_dir.join("#{@initialized_pid}.*.inserted_tables").to_s))
+    files = Dir.glob(@inserted_table_record_dir.join("#{@initialized_pid}.*.inserted_tables").to_s)
+
+    FileUtils.rm(files)
+    @logger&.debug "[MysqlRewinder] removed files: #{files.join(', ')}" if files.any?
```

Comment:
Fix for optimization

```suggestion
    @logger&.debug { "[MysqlRewinder] removed files: #{files.join(', ')}" } if files.any?
```